modules/installation-and-upgrade/pages/container-deployment/snippet-step-deploy-podman-certs.adoc

```diff
@@ -0,0 +1,32 @@
+. Execute one of the following commands, depending on the SSL certificate variant.
+  Replace `<FQDN>` with your fully qualified domain name of the {productname} Server:
++
+
+--
+* Using self-signed certificates provided by {productname}:
+
++
+
+[source,shell]
+----
+mgradm install podman <FQDN>
+----
+
+* With importing SSL certificates using third-party SSL certificate flags (the example needs to be completed with the other parameters if needed):
+
++
+
+[source,shell]
+----
+mgradm install podman <FQDN> \
+  --ssl-ca-intermediate <strings> \
+  --ssl-server-cert <string> \
+  --ssl-<name> <string> \
```

Comment:
For me this is confusing. What should I enter for <name>? 
If I read the comments, you properly mean another SSL parameter (e.g. CA), or do you mean my name?